system/CodeIgniter.php

```diff
@@ -141,6 +141,11 @@ class CodeIgniter
      */
     protected $useSafeOutput = false;
 
+    /**
+     * Context: 'web', 'php-cli', 'spark'
+     */
```

Comment:
```suggestion
     * @phpstan-property 'cli'|'spark'|'web'
     */
```

Reply:
Changed to `@phpstan-var`.

`@property` is for Magic properties.
See https://phpstan.org/writing-php-code/phpdocs-basics#magic-properties

Reply:
Ah thanks!